careers.html

```diff
@@ -438,6 +448,9 @@ <h2 class="text-block__heading like-h2">Can’t find your job here?</h2>
             $("#breadcrumbs").rcrumbs();
             
             // dropdown for vacancy
+			
+			$(window.location.hash).find('.dropdown__open').show(400);
```

Comment:
can you make the indentation consistent here?  it looks like the new lines you added are indented too much... you should also switch your tabs to be 'tabs as spaces' so that the file doesn't have mixed tabs and spaces for indentation